src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectFactory.Windows.cs

```diff
@@ -19,14 +19,14 @@ internal sealed class TdsParserStateObjectFactory
         //private static bool shouldUseLegacyNetorking;
         //public static bool UseManagedSNI { get; } = AppContext.TryGetSwitch(UseLegacyNetworkingOnWindows, out shouldUseLegacyNetorking) ? !shouldUseLegacyNetorking : true;
 
-#if DEBUG
+//#if DEBUG
```

Comment:
Is this to take the managed SNIPackat on windows? For test purpose?

Reply:
Yes, it allows me to switch modes for testing and profiling without having to manually edit the nuget file which is slow and tedious when you're trying to iterate on a problem. Since it's only active in debug builds which will never be in the published packages it shouldn't cause a problem.

Reply:
Wouldn't this be available in release builds now that you have commented out the ifdef?

Reply:
Yes. At some point it would be useful to re-enable the ability to switch modes on windows but I don't know when that point will be. I can reset this to be debug only again if you prefer.

Reply:
Yeah this should be debug only. I don't want this to be exposed in Release build unless we want to allow the product being shipped, to be able to do the switch with env variables. 
I definitely understand why you need this switch for your testing of release build and that is reasonable. But please exclude the changes from getting in Release build.

Reply:
Ok, done.